hbase/src/test/java/org/apache/metamodel/hbase/CreateTableTest.java

```diff
@@ -0,0 +1,209 @@
+            fail("Should get an exception that columnFamilies is empty");
+        } catch (IllegalArgumentException e) {
+            assertEquals("Can't create a table without having the tableName or columnFamilies", e.getMessage());
+        }
+    }
+
+    /**
+     * Goodflow. Create a table without the ID-Column, should work
+     *
+     * @throws IOException
+     */
+    @Test
+    public void testCreateTableWithoutIDColumn() throws IOException {
+        final HBaseTable table = createHBaseTable(TABLE_NAME, null, CF_FOO, CF_BAR, null);
+        final LinkedHashMap<HBaseColumn, Object> row = createRow(table, null, CF_FOO, CF_BAR, false);
+        final Set<String> columnFamilies = getColumnFamilies(getHBaseColumnsFromRow(row));
+        try {
+            final HBaseCreateTableBuilder hBaseCreateTableBuilder = (HBaseCreateTableBuilder) getUpdateCallback()
+                    .createTable(getSchema(), TABLE_NAME);
+
+            hBaseCreateTableBuilder.setColumnFamilies(columnFamilies);
+            hBaseCreateTableBuilder.execute();
+            checkSuccesfullyInsertedTable();
+        } catch (Exception e) {
+            fail("Should not get an exception (that the ID-column is missing)");
+        }
+    }
+
+    /**
+     * Goodflow. Create a table including the ID-Column (columnFamilies not in constructor), should work
+     */
+    @Test
+    public void testSettingColumnFamiliesAfterConstrutor() {
+        final HBaseTable table = createHBaseTable(TABLE_NAME, HBaseDataContext.FIELD_ID, CF_FOO, CF_BAR, null);
+        final LinkedHashMap<HBaseColumn, Object> row = createRow(table, HBaseDataContext.FIELD_ID, CF_FOO, CF_BAR,
+                false);
+        final Set<String> columnFamilies = getColumnFamilies(getHBaseColumnsFromRow(row));
+        try {
+            final HBaseCreateTableBuilder hBaseCreateTableBuilder = (HBaseCreateTableBuilder) getUpdateCallback()
+                    .createTable(getSchema(), TABLE_NAME);
+
+            hBaseCreateTableBuilder.setColumnFamilies(columnFamilies);
+            hBaseCreateTableBuilder.execute();
+            checkSuccesfullyInsertedTable();
+        } catch (Exception e) {
+            fail("Should not get an exception");
+        }
+    }
+
+    /**
+     * Goodflow. Create a table including the ID-Column (columnFamilies in constructor), should work
+     */
+    @Test
+    public void testCreateTableColumnFamiliesInConstrutor() {
+        final HBaseTable table = createHBaseTable(TABLE_NAME, HBaseDataContext.FIELD_ID, CF_FOO, CF_BAR, null);
+        final LinkedHashMap<HBaseColumn, Object> row = createRow(table, HBaseDataContext.FIELD_ID, CF_FOO, CF_BAR,
+                false);
+        final Set<String> columnFamilies = getColumnFamilies(getHBaseColumnsFromRow(row));
+        try {
+            getUpdateCallback().createTable(getSchema(), TABLE_NAME, columnFamilies).execute();
+            checkSuccesfullyInsertedTable();
+        } catch (Exception e) {
```

Comment:
Here too, don't catch the exception but have the method throw `IOException` instead.